package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.108",
   "description": "Type definitions for Corvid by Wix",
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint ./src/** ./test/** ./scripts/**",
```

Comment:
consider separating into a different commit

Reply:
Way so?
The lint command was busted, I fixed it cause I needed it

Reply:
what wasn't working before ?

Reply:
Nope 🤷‍♂️

Reply:
please add the lint script to executed as part of the test script as well:
`"test": "yarn lint && jest"`

Reply:
Nice, done